tests/unit_tests/model/objects/test_balance_pyo3.py

```diff
@@ -44,9 +44,9 @@ def test_account_balance_to_from_dict():
     assert account_balance == AccountBalance.from_dict(result_dict)
     assert result_dict == {
         "type": "AccountBalance",
-        "free": "1500000.00 USD",
-        "locked": "25000.00 USD",
-        "total": "1525000.00 USD",
+        "free": "1500000",
```

Comment:
The currency code is probably redundant in the string, but we should retain the implicit precision. The `to_str` method provides this (then we would get e.g.  `"1500000.00"`). The other choice is to use floats, which I don't think would be a better option?